app/Mail/PasswordResetNotification.php

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
Why is password reset already in this PR?
It should be in a different PR because it's another task.